commands/login.go

```diff
@@ -61,11 +63,12 @@ var loginCmd = &cobra.Command{
 
 		rest := newRest()
 		var token string
+		var key string
 
 		if providedAuthenticationMethod == "email" {
 			token = emailLogin(rest)
```

Comment:
This can be changed to `key` instead. Also it is probably to name the variable `accessKey`